src/mail/mail.service.ts

```diff
@@ -27,14 +37,38 @@ export class MailService {
         this.hostname = `${this.configService.getOrThrow('APP_HOST')}:${this.configService.getOrThrow('APP_PORT')}`;
     }
 
-    async sendUserConfirmation(user: User, token: string) {
+    async sendUserConfirmation(email: string, token: string) {
```

Comment:
It would be good to define what this function returns.
```suggestion
    async sendUserConfirmation(email: string, token: string): Promise<boolean> {
```